docs/status.md

```diff
@@ -14,9 +14,7 @@ One of the directions we're looking at is the have k0smotron working as a Cluste
 ## Known limitations
 
 Some of the areas we know have lot of shortcomings currently:
-
 - Control Plane configurability: As you know, k0s itself has lot of configurability, we plan to enable full configurability of the k0s control plane
-
 - Control plane exposing: Currently k0smotron only supports `NodePort` and `LoadBalancer` type services. While that is in itself quite ok quite often there's need to further configure e.g. annotations etc. on the created service to make them play nice with cloud provider implementations.
-
-- Updates: While k0smotron is able to update the cluster controlplane easily the update does not strecth into worker nodes.
\ No newline at end of file
+- Updates: While k0smotron is able to update the cluster controlplane easily the update does not strecth into worker nodes.
```

Comment:
Can we rebase this part on top of https://github.com/k0sproject/k0smotron/pull/485 that was merged yesterday?